README.md

```diff
@@ -130,7 +136,7 @@ If you wish to set the LogEntry `trace` property with a custom value, then set a
 
 ```js
 const winston = require('winston');
-const transport = require('@google-cloud/logging-winston');
+const transport = require('@google-cloud/logging-winston').LoggingWinston;
```

Comment:
I would favour an example optimizing for usability with Node 6+. I.e. destructuring assignment.

Reply:
Done